numpy/random/tests/test_extending.py

```diff
@@ -51,7 +51,9 @@ def test_cython(tmp_path):
                     reason="requires numba and cffi")
 def test_numba():
     from numpy.random._examples.numba import extending
+    assert extending is not None
 
 @pytest.mark.skipif(cffi is None, reason="requires cffi")
 def test_cffi():
     from numpy.random._examples.cffi import extending
+    assert extending is not None
```

Comment:
This too is code creep, but it seems harmless.

Reply:
ahh, no nvermind. It is needed to avoid the unused import warning

Reply:
Tempting to use an appropriate `# noqa` comment on these lines rather than adding a nonsensical test (why would this ever be `None`?)

Reply:
I've added `noqa` for now, but am interested in adding `main()`. For consistency, would this also need to be done for `cython/extending.pyx`? If so, this should possibly be best done with a separate PR for this suite of functions and tests.